vng_api_common/notifications/kanalen.py

```diff
@@ -35,10 +35,10 @@ def __repr__(self):
         cls_name = self.__class__.__name__
         return "%s(label=%r, main_resource=%r)" % (cls_name, self.label, self.main_resource)
 
-    def get_kenmerken(self, obj: Model) -> List[Dict]:
-        kenmerken = []
+    def get_kenmerken(self, obj: Model) -> Dict:
+        kenmerken = {}
```

Comment:
And now with the power of dict comprehensions :smile: 

```python
return {
    kenmerk: getattr(obj, kenmerk)
    for kenmerk in self.kenmerken
}
```

Reply:
fixed :)